File: RSPAClient/Core/AtPar.ViewModel/VM_MT_ATPAR_USER_PARAMS.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
   public class VM_MT_ATPAR_USER_PARAMS
    {
        public string PARAMETER_ID { get; set; }
        public string PARAMETER_VALUE { get; set; }
        public string LONG_DESCR { get; set; }
        public string SHORT_DESCR { get; set; }
        public string PARAMETER_TYPE { get; set; }
        public string VALIDATION { get; set; }
        public string MULTIPLE_VALUES { get; set; }
        public string REQUIRED_FLAG { get; set; }
        public string PROMPT_TABLE { get; set; }
        public string PROMPT_FIELD { get; set; }
        public string WHERE_CONDITION { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_MT_ATPAR_USER_PARAMS.ts
import { ATPAR_VALIDATION_RULES } from '../entities/atpar_validation_rules';
export class VM_MT_ATPAR_USER_PARAMS {

    public PARAMETER_ID: string;
    public PARAMETER_VALUE: any;
    public LONG_DESCR: string;
    public SHORT_DESCR: string;
    public PARAMETER_TYPE: string;
    public VALIDATION: string;
    public MULTIPLE_VALUES: string;
    public REQUIRED_FLAG: string;
    public PROMPT_TABLE: string;
    public PROMPT_FIELD: string;
    public WHERE_CONDITION: string;
    public validationRules: string;//ATPAR_VALIDATION_RULES[];
    public IsValidate: boolean;
    public DDLLIST: any;
    public TITLE: any;
}